src/variables/global_variables.cairo

```diff
@@ -0,0 +1,13 @@
+#[contract]
+mod GlobalExample {
+    // import the library function and types from the starknet core library 
+    use starknet::get_caller_address;
+    use starknet::ContractAddress;
+
+    #[external]
+    fn transfer(recipient: ContractAddress, amount: u256) {
```

Comment:
If your function doesn't perform a transfer, you shouldn't call it this way. You can give it a generic name like `foo` or `do_something`. Also, we don't need any parameters in this example.